iocore/net/quic/QUICConfig.h

```diff
@@ -156,3 +155,30 @@ class QUICConfig
 private:
   static int _config_id;
 };
+
+class QUICCertConfig
+{
+public:
+  static void startup();
+  static void reconfigure();
+  static SSLCertLookup *acquire();
+  static void release(SSLCertLookup *lookup);
+
+  using scoped_config = ConfigProcessor::scoped_config<QUICCertConfig, SSLCertLookup>;
+
+private:
+  static int _config_id;
+};
+
+class QUICMultiCertConfigLoader : public SSLMultiCertConfigLoader
+{
+public:
+  QUICMultiCertConfigLoader(const SSLConfigParams *p) : SSLMultiCertConfigLoader(p) {}
+
+  virtual SSL_CTX *default_server_ssl_ctx() override;
+  virtual SSL_CTX *init_server_ssl_ctx(std::vector<X509 *> &cert_list, const SSLMultiCertConfigParams *multi_cert_params) override;
```

Comment:
Does this have to be public?

Reply:
I'd like to make this private on #5032. But `SSLCreateServerContext()` needs `
SSLMultiCertConfigLoader::init_server_ssl_ctx()` public.

Reply:
Does it mean public `QUICMultiCertConfigLoader::init_server_ssl_ctx` will be needed in the future?

Reply:
Yes. If plugin needs to create SSL_CTX for quic, this might be exposed by TS API.